modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -1191,8 +1191,7 @@ public void testAddOrUpdateSystemObjectField() throws Exception {
 					systemObjectField.getExternalReferenceCode(),
 					modifiableSystemObjectDefinition.getObjectDefinitionId(),
 					ObjectFieldConstants.BUSINESS_TYPE_TEXT,
-					RandomTestUtil.randomString(),
-					RandomTestUtil.randomString(),
+					_generateTestDbColumnName(), _generateTestDbTableName(),
```

Comment:
I'm not sure if you noticed, but this scenario is about updating the `able` field. The `dbColumnName` and `dbTableName` are not updated in this operation, so you can pass `null` here.